README.md

```diff
@@ -35,7 +35,7 @@ First, complete the [basic Rust setup instructions](./docs/rust-setup.md).
 Use Rust's native `cargo` command to build and launch the template node:
 
 ```sh
-cargo run --release -- --dev --tmp
```

Comment:
why --tmp is removed?

Reply:
Upstream - https://github.com/substrate-developer-hub/substrate-node-template/commit/0fa941470d70c03169cedb08213fca6247c141f9